packages/main/src/TabContainer.js

```diff
@@ -122,20 +133,6 @@ const metadata = {
 			defaultValue: TabContainerTabsPlacement.Top,
 		},
 
-		/**
-		 * Defines whether the overflow select list is displayed.
-		 * <br><br>
-		 * The overflow select list represents a list, where all tab filters are displayed
-		 * so that it's easier for the user to select a specific tab filter.
-		 *
-		 * @type {boolean}
-		 * @defaultvalue false
-		 * @public
-		 */
-		showOverflow: {
```

Comment:
Keep the property and mark it as "deprecated".

Reply:
Done